lib/osml/osml_auth_server.ts

```diff
@@ -76,17 +45,17 @@ export interface OSMLAuthenticateProps {
 /**
  * Represents the construct that authenticates the user.
  */
-export class OSMLAuthenticate extends Construct {
+export class OSMLAuthServer extends Construct {
```

Comment:
I don't think this is the construct for the AuthServer (keycloak), is it?

Reply:
What should this be called?

Reply:
+1 to Jonathan. This construct is focusing on adding listener / rules to the ALB. So, we can rename something to like `OSMLAuth` or `OSMLAuthListenerRules`